SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -80,13 +81,37 @@
                 "p11-kit",
                 "patchelf",
                 "perl-JSON",
+                "picosat",
                 "pkgconf",
                 "pugixml",
                 "pycairo",
+                "python-async-generator",
+                "python-conda-package-handling",
+                "python-cpuinfo",
+                "python-cytoolz",
+                "python-fields",
+                "python-filelock",
+                "python-frozendict",
                 "python-google-auth",
                 "python-jwt",
                 "python-kubernetes",
+                "python-mock",
+                "python-nose",
+                "python-process-tests",
+                "python-pycosat",
+                "python-pytest-benchmark",
+                "python-pytest-cov",
+                "python-pytest-forked",
+                "python-pytest-mock",
+                "python-pytest-xdist",
                 "python-pywbem",
+                "python-responses",
+                "python-ruamel-yaml",
+                "python-ruamel-yaml-clib",
+                "python-sphinx-theme-py3doc-enhanced",
+                "python-toolz",
+                "python-tqdm",
+                "python3-pytest-asyncio",
```

Comment:
A question on the `python-xxx` packages.
Are they specifically python2 or they are a mix of both python2 & python3 ?